src/components/popups-etc/help.tsx

```diff
@@ -44,11 +78,35 @@ export default function Help(props: HelpProps) {
 
 			{tutorialHtml && props.persistenceState && visible.value && showingTutorial.value && (
 				<div class={styles.content} >
-					<div dangerouslySetInnerHTML={{ __html: tutorialHtml }} />
-					{props.persistenceState.value.kind === 'PERSISTED' && (
-						<Button onClick={() => {
-							props.showingTutorialWarning!.value = true
-						}} accent>Exit Tutorial</Button>
+					<div dangerouslySetInnerHTML ={{ __html: tutorialHtml }} />
+
+					{(props.persistenceState?.value.kind == "PERSISTED" || props.persistenceState?.value.kind == "SHARED") && (
+						<>
+							<br/>
+						<div class={styles.paginationContainer}>
+						{props.persistenceState.value.tutorialIndex != undefined
+							&& props.persistenceState.value.tutorialIndex > 0
+							&& (<Button onClick={previousPage} class={styles.paginationButton}><IoCaretBack/> Back</Button>)}
+
+							{props.persistenceState.value.tutorial
+								&& props.persistenceState.value.tutorial.length > 1
+									&& (<div class={styles.pageIndicator}>Page {(props.persistenceState.value.tutorialIndex || 0) + 1}/{props.persistenceState.value.tutorial && props.persistenceState.value.tutorial.length}</div>)}
+
+						{props.persistenceState.value.tutorialIndex != undefined
+							&& props.persistenceState.value.tutorial
+							&& props.persistenceState.value.tutorialIndex < props.persistenceState.value.tutorial.length - 1
+							&& (<Button onClick={nextPage} class={styles.paginationButton} accent>Next <IoCaretForward/></Button>)}
+
+
+							{props.persistenceState.value.tutorialIndex != undefined
+								&& props.persistenceState.value.tutorial
+								&& props.persistenceState.value.tutorialIndex == props.persistenceState.value.tutorial.length - 1
+								&& (<Button class={styles.paginationButton} onClick={() => {
+									props.showingTutorialWarning!.value = true
```

Comment:
```suggestion
									props.showingTutorialWarning?.value = true
```

the assertion might be code smell